ActindoWorkFlow/UserInterface/03a_Processes.md

```diff
@@ -102,11 +66,14 @@ Depending on the process status, the editing toolbar displays the following butt
     - **Aborted** (black)
 
 - *Suspended actions*  
-    If the process contains suspended actions, a ![Warning](../../Assets/Icons/Warning.png "[Warning]")(Warning) sign is displayed. Click the process to display the individual actions, see [Actions](#actions).
+    If the process contains suspended actions, a ![Warning](../../Assets/Icons/Warning.png "[Warning]")(Warning) sign is displayed. Click the process to display the suspended action. If desired, start the action to continue and finalize the process.
 
-- *Subject*  
-    Process description.
+- *Has a manual action*   
+    If the process contains a manual action, a ![Warning](../../Assets/Icons/Warning.png "[Warning]")(Warning) sign is displayed. Click the process to execute the manual action, see [Make a user decision](../Operation/09_TrackWorkflowProcess.md#make-a-user-decision).
 
```

Comment:
Make a user decision link geht hier nicht.
Ähnlich zu Beschriebung oben, Vorschlag: Click the process to display the manual action. If desired,...